worlds/terraria/Rules.dsv

```diff
@@ -1,153 +1,200 @@
 
 // Cavern
 Obsidian;
 Obsidian Skull;                     ;                                           Obsidian;
+Raider's Talisman;                  Calamity;                                   Obsidian;
 There are Some Who Call Him...;     Achievement | Grindy;
 Lava Charm;
 Demonite Ore;
-Demonite Bar;                       ;                                           Demonite Ore | (@calamity & #Calamity Evil Boss);
+Demonite Bar;                       ;                                           Demonite Ore;
 Evil Sword;                         ;                                           Demonite Bar;
+Coin of Deceit;                     Calamity;                                   Demonite Bar | Ruin Medallion;
 
 // Underground Ice
 Ice Skates;
-Flinx Fur Coat;                     ArmorMinions(1);
+Flinx Fur Coat;                     Armor Minions(1);
 
 // Underground Desert
+Stormlion Mandible;                 Calamity;
 Golfer;                             Npc;
+Party Girl;                         Npc;                                        @npc(14);
+Jolly Jamboree;                     Achievement | Grindy;                       Party Girl;
+Cool Slime;                         Npc;                                        Party Girl;
 
 // Sunken Sea
 Sea Prism;                          Calamity;
 Navyplate;                          Calamity;                                   Sea Prism & Obsidian;
 
 // Underground Jungle
 Anklet of the Wind;
+Feral Claws;
 Stinger;
 Jungle Spores;
 Vine;
 Blade of Grass;                     ;                                           Stinger & Jungle Spores & Vine;
+Nature's Gift;
+Bezoar;
 Summoning Potion;                   Minions(1);
 
+// The Aether
+A Shimmer In The Dark;              Achievement;
+
 // Underworld
 It's Getting Hot in Here;           Achievement;
 Rock Bottom;                        Achievement;
 Obsidian Rose;
 Havocplate;                         Calamity;
+Magma Stone;
 
 // Evil
 Smashing, Poppet!;                  Achievement;
 Arms Dealer;                        Npc;
 Leading Landlord;                   Achievement;                                Nurse & Arms Dealer; // The logic is way more complex, but that doesn't affect anything
 Completely Awesome;                 Achievement;                                Arms Dealer;
+Illegal Gun Parts;                  ;                                           Arms Dealer | Flamethrower;
+
+// Abyss
+Ink Bomb;                           Calamity;
 
 // King Slime
 King Slime;                         Location | Item;
-Sticky Situation;                   Achievement | Grindy;
+Sticky Situation;                   Not Getfixedboi | Achievement | Grindy;
 The Cavalry;                        Achievement;
 Solidifier;                         ;                                           #King Slime;
+Nerdy Slime;                        Npc;                                        King Slime;
```

Comment:
Nerdy Slime appears as soon as King Slime is defeated, so only killing him is required, not the flag.